src/preprocess.js

```diff
@@ -124,7 +124,7 @@ function preprocessPlantUmlIncludes (diagramText, dirPath, includeOnce, includeS
           const target = parseTarget(args[1])
           const urlSub = target.url.split('!')
           const trailingContent = target.comment
-          const url = urlSub[0].replace(/\\ /g, ' ')
+          const url = urlSub[0].replace(/\\ /g, ' ').replace(/(\r\n|\n|\r|\s*)$/mg, '')
```

Comment:
I believe it can be simplified to:

```suggestion
          const url = urlSub[0].replace(/\s/g, ' ')
```

Reply:
Hello @Mogztter ,
when you change to what you have suggested, there are several tests that are failing.

![image](https://user-images.githubusercontent.com/43267376/182090613-2e13d6b2-95af-479b-ac80-a5b00ec31820.png)

![image](https://user-images.githubusercontent.com/43267376/182090884-ed69de84-4fcb-4198-85ca-706c859aac4c.png)

Best Regards

Reply:
Indeed, we should only remove trailing space characters:

```js
urlSub[0].replace(/\s+$/g, ' ')
```

Reply:
I don't remember exactly why we need `.replace(/\\ /g, ' ')` but I think we should keep it for now

Reply:
Hello @Mogztter,
I changed as per your advice to 
`const url = urlSub[0].replace(/\\ /g, ' ').replace(/\s+$/g, '')`
and did some more tests ... also on the project where I identified the issue and where I have the complete context.
All tests passed successfully.
Thanks and Best Regards

Reply:
Hello @Mogztter,
sorry for asking since I never contributed to an open source project via pull request.
What is the next thing I need to do?
Is there any further action needed by my side?

Thanks

Reply:
Hello @mijlouis no it's all good now, I will merge it, thanks for your contribution 🤗